src/main/resources/templates/ruby/test/test/test_behavior.rb

```
@@ -0,0 +1,64 @@
+
+require_relative "test_helper"
+
+require "date"
+
+class TestBehavior < Minitest::Test
+  def setup
+    @dates = [
+      Date.new(2018, 11, 8),
+      Date.new(2017, 4, 15),
+      Date.new(2016, 2, 15),
+      Date.new(2017, 9, 15)
```

rubocop (1.69.1)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)